packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,260 +1,428 @@
+                gapped
+                preset="default"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                border-color="primary"
+                gapped
+                preset="primary"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                border-color="primary"
+                gapped
+                preset="secondary"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Rounded</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                border-color="primary"
+                gapped
+                rounded
+                preset="default"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                border-color="primary"
+                gapped
+                rounded
+                preset="primary"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                border-color="primary"
+                gapped
+                rounded
+                preset="secondary"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Custom active page color</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                preset="default"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                preset="primary"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                preset="secondary"
+                active-page-color="warning"
+                gapped
+                rounded
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Sizes</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                size="small"
+              />
+              <va-pagination
                 v-model="activePage"
+                :pages="5"
+                size="medium"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                size="large"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Colored Pagination</td>
+          <td>Colored</td>
           <td>
             <VbCard>
               <va-pagination
```

Comment:
I would assume colored would cover cursor color as well. The way it works for button etc.

<img width="467" alt="image" src="https://user-images.githubusercontent.com/5394573/189894064-b89f64fc-e426-4ea9-8451-84c2c016e866.png">